src/EditorFeatures/TestUtilities/Classification/AbstractClassifierTests.cs

```diff
@@ -1,176 +1,298 @@
         }
 
-        [DebuggerStepThrough]
-        protected Tuple<string, string> Event(string value)
+        protected Task TestInMethodAsync(
+            string methodName,
+            string code,
+            params FormattedClassification[] expected)
         {
-            return ClassificationBuilder.Event(value);
+            return TestInMethodAsync("C", methodName, code, expected);
         }
 
-        [DebuggerStepThrough]
-        protected Tuple<string, string> String(string value)
+        protected Task TestInMethodAsync(
+            string code,
+            ParseOptions[] parseOptionsSet,
+            params FormattedClassification[] expected)
         {
-            return ClassificationBuilder.String(value);
+            return TestInMethodAsync("C", "M", code, parseOptionsSet, expected);
         }
 
-        [DebuggerStepThrough]
-        protected Tuple<string, string> Verbatim(string value)
+        protected Task TestInMethodAsync(
+            string code,
+            params FormattedClassification[] expected)
         {
-            return ClassificationBuilder.Verbatim(value);
+            return TestInMethodAsync("C", "M", code, expected);
         }
 
-        [DebuggerStepThrough]
-        protected Tuple<string, string> Keyword(string value)
+        protected async Task TestInNamespaceAsync(
+            string code,
+            ParseOptions[] parseOptionsSet,
+            params FormattedClassification[] expected)
         {
-            return ClassificationBuilder.Keyword(value);
+            var allCode = WrapInNamespace(code);
+
+            await TestAsync(code, allCode, parseOptionsSet, expected);
         }
 
-        [DebuggerStepThrough]
-        protected Tuple<string, string> PPKeyword(string value)
+        protected async Task TestInNamespaceAsync(
+            string code,
+            params FormattedClassification[] expected)
         {
-            return ClassificationBuilder.PPKeyword(value);
+            var allCode = WrapInNamespace(code);
+
+            await DefaultTestAsync(code, allCode, expected);
         }
 
         [DebuggerStepThrough]
-        protected Tuple<string, string> PPText(string value)
-        {
-            return ClassificationBuilder.PPText(value);
-        }
+        protected static FormattedClassification[] Classifications(params FormattedClassification[] expected) => expected;
 
         [DebuggerStepThrough]
-        protected Tuple<string, string> Identifier(string value)
-        {
-            return ClassificationBuilder.Identifier(value);
-        }
+        protected static ParseOptions[] ParseOptions(params ParseOptions[] options) => options;
 
         [DebuggerStepThrough]
-        protected Tuple<string, string> Inactive(string value)
-        {
-            return ClassificationBuilder.Inactive(value);
-        }
+        protected FormattedClassification Struct(string text) => ClassificationBuilder.Struct(text);
```

Comment:
Would a `using static` of `ClassificationBuilder` avoid the need to have all of these?

Reply:
heh-heh. Yes, I think you're right. I'll take a look at that.